extensions/wikia/TrackingOptIn/styles/ext.wikia.trackingSettingsManager.css

```diff
@@ -0,0 +1,7 @@
+#privacy-settings-button {
+    cursor: pointer;
+    display: block;
+    margin: 24px auto;
+    text-align: center;
+    width: 50%;
```

Comment:
IMO looks better without expanding it to 50% ;)

Reply:
due to display block it will then take full article width